src/Shared/RoslynUtils/WellKnownTypes.cs

```diff
@@ -86,6 +86,26 @@ private INamedTypeSymbol GetAndCache(int index)
         return _lazyWellKnownTypes[index]!;
     }
 
+    // Look for types within well-known assemblies only.
```

Comment:
This comment is wrong. We're prioritizing types in well-known assemblies so if a customer defines their own type, ours is used.

Reply:
Fair enough, did a quick language tool to indicate that this _filters_ for types within well-known assemblies. I think that is a little more obvious.